components/keys/Cargo.toml

```diff
@@ -8,6 +8,7 @@ publish = false
 byteorder = "1.2"
 derive_more = "0.15.0"
 failure = "0.1"
+farmhash = "1.1.5"
```

Comment:
This should also exist in Cargo.toml in the project root. Is that one still used? If not, that can be removed.

Reply:
Good catch, removed.